two_encoder_module/two_encoder_src/two_encoder_src.ino

```diff
@@ -435,7 +440,12 @@ void setup()
     while (!Serial)
     ;
 #endif
-
+  
+    // this is necessary for reconnecting a device because it need some timme for the serial port to get open, but meanwhile the handshake message was send from eos
+    SLIPSerial.beginPacket();
+    SLIPSerial.write((const uint8_t*)HANDSHAKE_REPLY.c_str(), (size_t)HANDSHAKE_REPLY.length());
+    SLIPSerial.endPacket();
+ 
```

Comment:
Out of curiosity, did you experience this issue? Does it only happen with native USB serial? We tested only with Arduino Uno.